test/framework/namespace.go

```diff
@@ -70,6 +70,7 @@ func OperateOnNamespaceIfExists(namespace string, operate func(namespace string)
 	} else if ok {
 		return operate(namespace)
 	}
+	GetLogger(namespace).Warn("Namespace not found for operation")
```

Comment:
Maybe even better would be to name which namespace was not found, but maybe that is too much. :)

Reply:
well namespace is in the logger already `GetLogger(namespace)`

Reply:
True :)

Reply:
Well, namespace name is already reflected in logger name